packages/api/src/external/commonwell/organization.ts

```diff
@@ -76,6 +76,29 @@ export async function organizationToCommonwell(
   return cwOrg;
 }
 
+export const get = async (oid: string): Promise<CWOrganization | undefined> => {
```

Comment:
Let's use normal functions, not arrow functions, please - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#b669ec3473f1414b9d234cbdac20041b).

Reply:
Also, can we update `getOne` on `packages/api/src/external/commonwell/__tests__/organization.ts` so it uses this one, please? And mark it as `@deprecated`. This way you don't need to update all the places that use it.

```ts
export async function getOne = cwCommands.organization.get;
```

Reply:
Its only used in one place so ill just remove it